pkg/controller/directvolumemigration/rsync.go

```diff
@@ -1244,62 +1273,57 @@ func getMD5Hash(s string) string {
 	return hex.EncodeToString(hash[:])
 }
 
-func (t *Task) haveRsyncClientPodsCompletedOrFailed() (bool, bool, error) {
+// reportRsyncPodProgress reads DVMP CR present for all PVCs and generates progress information in CR status
+func (t *Task) reportRsyncPodProgress() (bool, error) {
```

Comment:
the function name does not suggest we are returning a bool. rename to `haveAllRsyncOperationsCompleted` instead?

Reply:
+1, updating